requirements.txt

```diff
@@ -8,4 +8,4 @@ pytest >= 2.8.5
 PyYAML >= 3.11
 scipy >= 0.17.1
 wcsaxes
-
+-e git+git://github.com/pynbody/pynbody@master#egg=pynbody
```

Comment:
@AnthonyHorton  @wtgee 
I just did the same as we have done about `gunagal`, but it does not work here. Please guide me on how to add `pynbody` to the `requirements.txt`.

Reply:
Suggest google'ing around for an answer... this should be straight-forward as lots of poeple do it.